command/scripting.go

```diff
@@ -436,3 +523,27 @@ func (c *scriptingExec) Execute(ctx context.Context, comm client.Communicator, l
 
 	return catcher.Resolve()
 }
+
+func (c *scriptingExec) reportTestResults(ctx context.Context, comm client.Communicator, logger client.LoggerProducer, conf *model.TaskConfig, report io.Reader) error {
+	switch c.Harness {
+	case "go", "golang":
+		// kim: TODO: I don't actually know if the test name is consequential,
+		// but I included it because if  I didn't specify it, the REST request
+		// to post logs hangs and the task times out.
+		log, result, err := parseTestOutput(ctx, conf, report, "output.test")
```

Comment:
Is the test output file configurable? I think we name it like this, but I don't believe go test has any conventions

Reply:
It's not actually an output file. The output log is stored entirely in-memory (`report`), so I just used a placeholder name for the test name. I couldn't figure out how the name in test results was used, only that it had to be populated for the REST request to post the test logs to succeed.